CDAP-12137 add null field splitter

albertshau opened the pull request:
Adding a SplitterTransform that has two output ports, one for
records with a null field value and one for records that have
a non-null value.

albertshau committed:
CDAP-12137 add null field splitter

Adding a SplitterTransform that has two output ports, one for
records with a null field value and one for records that have
a non-null value.

gsps1 commented on transform-plugins/src/main/java/co/cask/hydrator/plugin/NullFieldSplitter.java:
```diff
@@ -0,0 +1,163 @@
+   */
+  public static class Conf extends PluginConfig {
+    @Description("Which field should be checked for null values.")
+    private final String field;
```
should we allow this to be a macro ?

albertshau replied:
can make it macro enabled, will update.

gsps1 reviewed (commented):
just one question, LGTM otherwise.

albertshau committed:
CDAP-12137 macro enable field property

Macro enabling the field property and adding an endpoint to get
the output schemas.

gsps1 commented:
LGTM
